test_checkoff__task.py

```diff
@@ -0,0 +1,30 @@
+from task_manager_ import TaskManager
+
+import unittest
+
```

Comment:
from unittest.mock import patch

Currently, test_already_completed_task and test_invalid_task_number lack assertions I think, which would make them incomplete. Consider using unittest.mock to capture printed output and verify it.